tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to the Advanced Research Computing (ARC)\n",
+    "\n",
+    "Open a terminal and remotely connect to ARCUS using ssh (secure shell), using your username:\n",
+    "\n",
+    "`% ssh -X username@oscgate.arc.ox.ac.uk`\n",
```

Comment:
You're not using X-forwarding in the tutorial, can we drop the `-X` flags?

Reply:
Done